src/hardware/serialport/serialport.cpp

```diff
@@ -1368,8 +1368,7 @@ void SERIAL_Destroy(Section *sec)
 }
 
 void SERIAL_Init (Section * sec) {
-	// should never happen
-	if (testSerialPortsBaseclass) delete testSerialPortsBaseclass;
+	delete testSerialPortsBaseclass;
```

Comment:
Cleans up a PVS warning about unnecessarily checking before deleting (because `delete`ing a nullptr is fine).